### PR TITLE
Architecture update

### PR DESCRIPTION
**Pull request by pfarnach**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,38 +1,53 @@
...
-  "main": "server.js",
+  "name": "resistance-calendar-frontend",
+  "version": "0.1.0",
   "license": "MIT",
+  "scripts": {
+    "build": "rimraf dist && webpack -p",
+    "dev": "webpack-dev-server",
+    "test": "jest --config=testConfig/jest.config.json --watch",
+    "lint": "clear && eslint ./src --ext .js --cache",
+    "lint:watch": "onchange ./src --initial -- yarn run lint"
```
Comment (pdw207) [inline package.json]: Can we add a script for npm users or pass in a variable?

File: package.json [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
-  "version": "1.0.0",
-  "repository": "https://github.com/ResistanceCalendar/resistance-calendar-frontend.git",
-  "main": "server.js",
+  "name": "resistance-calendar-frontend",
+  "version": "0.1.0",
   "license": "MIT",
+  "scripts": {
+    "build": "rimraf dist && webpack -p",
+    "dev": "webpack-dev-server",
+    "test": "jest --config=testConfig/jest.config.json --watch",
```
Comment (pdw207) [inline package.json]: Do we need a test:single for TravisCI?

File: webpack/vendorRegistry.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module.exports = [
+  'axios',
+  'react',
+  'react-dom',
+  'lodash',
```
Comment (pdw207) [inline webpack/vendorRegistry.js]: Are we using lodash?

Comment (dorono) [inline webpack/vendorRegistry.js]: @pdw207 - I find lodash pretty handy for some of the more complex operations not handled by ES6 out of the box, so I, for one, wouldn't mind keeping it in unless anyone is opposed

File: package.json (hunk 2 above)
Comment (pfarnach) [inline package.json]: Good point.  To be consistent with the linting scripts, would `test` and `test:watch` work?

Comment (pfarnach) [inline package.json]: Updated

File: package.json (hunk 1 above)
Comment (pfarnach) [inline package.json]: Updated

**Pull request merged**